pdsApp/src/app/models/Advisor.java

```diff
@@ -96,4 +106,52 @@ public boolean getError() {
     return error;
   }
 
+  public String getErrorMessage() {
+    return errorMessage;
+  }
+
+  public void getAdvisor() {
+    Connection conn = null;
+    Statement stat = null;
+    ResultSet results = null;
+    
+    try {
+      conn = Server.getConnection();
+      stat = conn.createStatement();
+      results = stat.executeQuery("SELECT * FROM t_advisor WHERE login = '" + this.login + "' AND password = '" + this.password + "';");
+      
+      results.next();
+
+      this.id = results.getInt("id_advisor");
+      this.last_name = results.getString("last_name");
+      this.first_name = results.getString("first_name");
+      this.director = results.getBoolean("director");
+      this.id_agency = results.getInt("id_agency");
+      this.password = results.getString("password");
+      this.login = results.getString("login");
+      this.mail = results.getString("mail");
+      
+      results.close();
+      
+      results = stat.executeQuery("SELECT t_adress.id_adress, t_adress.street_nb, t_adress.street_name, t_adress.city_name, t_adress.zip_code " + 
+                                  "FROM t_adress, t_advisor " + 
+                                  "WHERE t_adress.id_adress = t_advisor.id_agency AND t_advisor.login = '" + this.login + "' AND t_advisor.password = '" + this.password + "';");
```

Comment:
Are you trying to get the agency address? If so, you don't need to check the login and pwd here.